src/docs/concepts-messages/email-notifications/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Email notifications
+slug: email-notifications
+updated: 2022-07-21
+category: concepts-messages
+ingress:
+  Email notifications are messages users receive when something
+  important happens on the marketplace (e.g., the user has received a
+  message or a payment on the platform). Notifications get delivered via
+  email.
+published: true
+---
+
+Sharetribe Flex sends email notifications to users when specific events
+occur in your marketplace. There are two categories of email
+notifications in Flex: built-in email notifications, which relate to
+user account management, and transaction process emails, which get
+triggered at specific stages of transactions. For instance, an built-in
+email notification gets sent when a user changes their email or password
+or when they need to verify a new email address. A transaction process
+email can inform the user of a successful payment or a new booking
+request.
+
+## Enable email notifications
+
+Email notifications are automatically enabled in your test marketplace.
+However, in your production marketplace, you must
+[configure outgoing email settings](https://www.sharetribe.com/docs/how-to/set-up-outgoing-email-settings/)
+for email notifications to work.
+
+Users won't receive email notifications until they have verified their
```

Comment:
Do we have a Flex Docs policy on using abbreviations vs full expressions (can't vs can not etc)? I tend to avoid abbreviations when writing Docs text, but I don't think we have an overarching decision on whether to use them.

Reply:
Good point. I'll stick to non-abbreviations from now on!